docs/markdown/Features-module.md

```diff
@@ -0,0 +1,320 @@
+# Features module
+
+## Overview
+
+Dealing with a numerous of CPU features within C and C++ compilers poses intricate challenges,
+particularly when endeavoring to support an extensive set of CPU features across diverse
+architectures and multiple compilers. Furthermore, the conundrum of accommodating
+both fundamental features and supplementary features dispatched at runtime
+further complicates matters.
+
+In addition, the task of streamlining generic interface implementations often leads to the necessity
+of intricate solutions, which can obscure code readability and hinder debugging and maintenance.
+Nested namespaces, recursive sources, complex precompiled macros, or intricate meta templates
+are commonly employed but can result in convoluted code structures.
+
+Enter the proposed module, which offers a simple pragmatic multi-target solution to
+facilitate the management of CPU features with heightened ease and reliability.
```

Comment:
This section reads more like an MR proposal, it feels like it's trying to justify the existence of the module. I don't know if that makes sense, and I'm not really sure how to improve it